static/js/click_handlers.js

```diff
@@ -581,25 +592,12 @@ export function initialize() {
                 .find(".user-presence-link");
             const user_id_string = elem.attr("data-user-id");
             const title_data = buddy_data.get_title_data(user_id_string, false);
-            do_render_buddy_list_tooltip(elem, title_data);
-        },
-    );
-
-    $("#user_presences").on(
-        "mouseleave click",
-        ".user-presence-link, .user_sidebar_entry .user_circle, .user_sidebar_entry .selectable_sidebar_block",
-        (e) => {
-            e.stopPropagation();
-            const elem = $(e.currentTarget)
-                .closest(".user_sidebar_entry")
-                .find(".user-presence-link");
-            $(elem).tooltip("destroy");
+            do_render_buddy_list_tooltip(elem.parent(), title_data);
```

Comment:
I can reproduce a bug with this widget like this:

![image](https://user-images.githubusercontent.com/2746074/113949259-b1a92000-97c3-11eb-825f-79e33e6b3316.png)

Reproducer:
* Hover over various users in a normal size window
* Make window narrow
* Open users panel
* Hover over users

I'm not sure why, but it seems like we're effectively ending up registering two tippyjs widgets with different configuration on the same object.

Reply:
We were creating a new instance on every mouseover but not destroying the old one. Thanks for noticing this.